src/ReceiveSubcommands.cc

```diff
@@ -961,7 +962,14 @@ static bool drop_item(
     // If the game is not BB, forward the request to the leader instead of
     // generating the item drop command
   } else {
-    return false;
+      if (s->drops_enabled == true)

```

Comment:
code convention in this repo seems to be that `{` is on the same line as the `if`